api/provider/provider.go

```diff
@@ -43,9 +44,9 @@ type Provider interface {
 
 	LogStream(app string, w io.Writer, opts structs.LogStreamOptions) error
 
-	ReleaseDelete(app, id string) (*structs.Release, error)
+	ReleaseBatchDelete(app, buildID string) error
```

Comment:
Can we get away without exposing `ReleaseBatchDelete` in the provider interface?